table/block_based/filter_policy.cc

```diff
@@ -756,4 +756,34 @@ FilterBuildingContext::FilterBuildingContext(
 
 FilterPolicy::~FilterPolicy() { }
 
+Status FilterPolicy::CreateFromString(
```

Comment:
You mean give the example of the input string "Value" and also how and when to use the CreateFromString in the comments.